.env.example

```diff
@@ -22,6 +22,13 @@ NODE_ENV="development"
 # Only used for development purpose
 # PORT="3000"
 
+##################################################
+# Tell-me
+TELL_ME_URL=""
+TELL_ME_TOKEN=""
+TELL_ME_CONTRIBUTOR_SURVEY_ID=""
```

Comment:
Pour les clés des champs du formulaire, ça me paraît un peu too much de les mettre dans des ENVs même si je suis d'accord que toute configuration personnalisable devrait aller là.

Un entre-deux serait peut-être de simplement créer une seule ENV var pour ça qui soit une KV pair en JSON string ?

Reply:
On peut ; j'ai vraiment fait au plus simple au début ; et je pensais améliorer au fur et à mesure

J'ai tendance à être très itératif, pour éviter les grosses PR :)